packages/firebase/package.json

```diff
@@ -42,6 +42,7 @@
     "@firebase/database": "0.4.0",
     "@firebase/firestore": "1.3.0",
     "@firebase/functions": "0.4.6",
+    "@firebase/installations": "0.1.0",
```

Comment:
out of curiosity:
when do we plan to go to "1.0.0"?

Reply:
Explained [here](http://doc/1jWfkOoxk2mtjD0H62SUxFfAAm4c8GXSdLvIkdwDC6b4#heading=h.kstmuj6ckw41).